.github/workflows/pull-requests.yml

```diff
@@ -83,6 +83,44 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
+  java-compatibility:
```

Comment:
Can you please:
1. rename it to `compatibility-test` as intention more to see the difference in behaviour of `PostgreSQL` and `database` rather then see if `database` supports `java-pgjdbc-client`.
2. remove same job from `merge.yaml` file?

Reply:
>     1. rename it to `compatibility-test` as intention more to see the difference in behaviour of `PostgreSQL` and `database` rather then see if `database` supports `java-pgjdbc-client`.

Sure. I will rename to `compatiblity-test`.

> 
>     2. remove same job from `merge.yaml` file?

It seems that it is used for testing the new docker image (not only the code on master).

And `pull_requests.yml` tests the `dev` build by `cargo build` currently. I think it could save some compile time than the `release` one. Should we leave `release` build in latest docker image and testing in `merge.yaml`?
WDYT?

Reply:
> It seems that it is used for testing the new docker image (not only the code on master).


I haven't think of it as testing docker `:latest`. I configured it that because I don't know about trick that you used in `ci/erlang_client.sh` with `cargo build` and `&` 😄 

this what I think should be removed 
https://github.com/alex-dukhno/database/blob/master/.github/workflows/merge.yaml#L26-L62
and dependency on `compatibility-test` in `merge.yaml` https://github.com/alex-dukhno/database/blob/master/.github/workflows/merge.yaml#L66

latest docker image should be published

Reply:
Hi @alex-dukhno , I remove `compatibility-test` from `merge.yaml`. Please help review. Thanks.